resources/js/components/NavBar/DesktopNav.tsx

```diff
@@ -20,7 +20,7 @@ export default function DesktopNav() {
     }
 
     return (
-        <nav className="hidden lg:flex items-center justify-around bg-pink-500 text-white px-10 font-bolder">
+        <nav className="z-50 hidden lg:flex items-center justify-around bg-pink-500 text-white px-10 font-bolder">
```

Comment:
generally using maximum z index is not a good idea, try to use as minimum as possible